core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java

```diff
@@ -37,7 +42,12 @@ public void accept(Event<ClusterStopRequest> event) {
         ClusterStopResult result;
         try {
             Stack stack = stackService.getByIdWithListsInTransaction(request.getResourceId());
-            apiConnectors.getConnector(stack).stopCluster(false);
+            boolean clusterManagerRunning = apiConnectors.getConnector(stack).clusterStatusService().isClusterManagerRunning();
```

Comment:
as I see only `ApiException` is handled here. Are we sure we can't receive any other type of exception when CM is not running?

Reply:
indeed, I'm checking that

Reply:
I have stopped the CM and working fine, but I didn't check the exception

Reply:
have you tested it with clusterproxy/CCM?

Reply:
Nit yet, I will try to test it

Reply:
Answer for the first question:
We can receive an ApiException here, the ApiException store the status code like 502

Reply:
Working with cluster proxy as well

Reply:
dh works as well